.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "standard"
+};
```

Comment:
Tiny nitpick: generally our frontend projects have a newline at the end of the file. This won't block accepting the PR, but adding a newline would be appreciated :)